src/components/Main/main.css

```diff
@@ -3,9 +3,12 @@ and then the modifier is anything that modifies the element (e.g disabled)
 
 <label className="Main__label-title Main__label-title--row">Drama</label> */
 
-.center {
-  margin: 0 auto;
-  width: fit-content;
+.main {
+  margin: auto;
+  left:0;
+  right: 0;
+  top: 0;
+  bottom: 0;
```

Comment:
I dont think these are doing anything

Reply:
We should add some padding though, the movies are right up against the left side, lets find out the padding netflix uses. 
<img width="943" alt="Screen Shot 2022-02-03 at 10 40 02 PM" src="https://user-images.githubusercontent.com/60586549/152468128-85a566ea-8673-4d9e-a63c-73f9c70b5d6f.png">
.